src/components/atoms/reference/reference.tsx

```diff
@@ -0,0 +1,37 @@
+import { Author, Reference as ReferenceData } from '../../../types';
+import './reference.scss';
```

Comment:
Let's add a test for this. As this is a refactor it should be easy to use the test for reference-list as a starting point.

Reply:
This is not ready for review as yet

Reply:
ok, that's fine. stepping away.